config_src/mct_driver/mom_ocean_model_mct.F90

```diff
@@ -520,10 +520,10 @@ subroutine update_ocean_model(Ice_ocean_boundary, OS, Ocean_sfc, &
     endif
     if (OS%icebergs_alter_ocean)  then
       if (do_dyn) &
-        call iceberg_forces(OS%grid, OS%forces, OS%use_ice_shelf, &
+        call iceberg_forces(OS%grid, OS%US, OS%forces, OS%use_ice_shelf, &
```

Comment:
Looks like OS%US argument here was added inadvertently. It needs to be removed.

Reply:
@hallberg-noaa most likely added this to enable dimensional scaling testing.  He probably ought to be involved in the conversation if it needs to be removed.

Reply:
The equivalent call in [coupled_driver](https://github.com/NOAA-GFDL/MOM6/blob/dev/gfdl/config_src/coupled_driver/ocean_model_MOM.F90#L510) does not have this argument. We will wait for @Hallberg-NOAA's confirmation before deleting it.

Reply:
I think @Hallberg-NOAA  meant to add the OS%US argument to iceberg_**fluxes**() calls, and NOT iceberg_**forces**() , which doesn't expect a unit scaling argument.

Reply:
Also, we need to add the OS%US argument to iceberg_fluxes() call in line 557. After making both changes, all CESM tests have passed. @gustavo-marques , once @Hallberg-NOAA confirms, I can push both changes and merge this.

Reply:
Sounds good, thanks @alperaltuntas.

Reply:
Yes, the correction that @alperaltuntas suggests is the right one.  This OS%US argument should only have been added to iceberg_fluxes() and not to iceberg_forces().  Please go ahead and make the change to the mct coupler code as suggested.